src/components/chats/hooks/useMessagesCount.ts

```diff
@@ -0,0 +1,44 @@
+import { getCommentIdsByPostIdFromChainQuery } from '@/services/subsocial/commentIds'
+import { getPostMetadataQuery } from '@/services/subsocial/datahub/posts/query'
+import { getDatahubConfig } from '@/utils/env/client'
+
+export function useMessagesCount(chatId: string) {
+  const isDatahubEnabled = !!getDatahubConfig()
```

Comment:
Here isDatahubEnabled in previous hooks - canUseDataHub. Please use one name for all hooks